### PR TITLE
use pulp 3.6 for katello 3.17

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
note, we may want to wait for rc2 to merge this

**Comment by evgeni** [comment]
@jlsherrill is this still needed? I'd assume so.

We have a test that for 3.17 we don't set a pulpcore version:
https://github.com/theforeman/forklift/blob/d55e8ce27144196e365bb7b99067d9c72315a823/roles/forklift_versions/molecule/default/verify.yml#L11-L25

This will need to be adjusted (probably just flipping it to Foreman 2.1/Katello 3.16 is the simplest here)

File: vagrant/config/versions.yaml [hunk 1]
```diff
@@ -79,6 +79,7 @@ installers:
   - foreman: '2.2'
     katello: '3.17'
     pulp: '2.21'
+    pulpcore: '3.6'
```
Comment (ekohl) [inline vagrant/config/versions.yaml:82]: By now this should be 3.7, right?

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by evgeni** [comment]
#1222 should fix that all :)

**Comment by evgeni** [comment]
closing given #1222 is merged.

**Pull request closed without merging**